go/client/cmd_prove.go

```diff
@@ -92,13 +104,32 @@ func (p *CmdProve) Run() error {
 		return err
 	}
 
-	if p.listServices {
+	if p.listAllServices {
 		services, err := cli.ListProofServices(context.TODO())
 		if err != nil {
 			return err
 		}
 		ui := p.G().UI.GetTerminalUI()
-		ui.Printf("Supported services are: %s.\n", strings.Join(services, ", "))
+		ui.Printf("All supported services:\n")
+		for _, service := range services {
+			ui.Printf("  %s\n", service)
+		}
+		if len(services) > 50 {
```

Comment:
whats this here for?

Reply:
If the list even becomes short it would be weirder to say "the end" after it.